angular/src/app/modules/delivery-management/delivery/available-resource-tab/bill-account-plan/bill-account-plan.component.html

```diff
@@ -153,6 +153,32 @@
                 (click)="applyProjectStatusFilter()"
               ></i>
             </div>
+            <div class="mt-1 d-flex align-items-center mb-3 ml-2">
+              <mat-form-field
+                appearance="outline"
+                class="flex-grow-1"
+                style="width: 180px"
+              >
+                <mat-label>Charge Filter</mat-label>
+                <mat-select
+                  placeholder="Select charge"
+                  [(ngModel)]="isChargeText"
+                  (selectionChange)="onIsChargeFilter()"
+                >
+                  <mat-option
+                    *ngFor="let item of isChargeList"
+                    [value]="item.text"
+                    >{{ item.text }}</mat-option
+                  >
+                </mat-select>
+              </mat-form-field>
+              <i
+                style="margin-top: 0 !important"
+                class="fas fa-times fa-lg ml-2 mt-2"
+                *ngIf="isChargeText"
```

Comment:
*ngIf="isChargeText"
=> *ngIf="isShowBtnClearChargeFilter()"

isShowBtnClearChargeFilter() {
 return this.isCharge != null;
}